base/docs/helpdb/Base.jl

```diff
@@ -7376,7 +7376,7 @@ for the effects of compilation.
     Array(dims)
 
 `Array{T}(dims)` constructs an uninitialized dense array with element type `T`. `dims` may
-be a tuple or a series of integer arguments. The syntax `Array(T, dims)` is also available,
+be a tuple or a series of integer arguments. The syntax `Array{T}(dims)` is also available,
```

Comment:
You don't want to change this one.  It's interesting that it's documented as deprecated, but I guess we never actually made it show a deprecation warning.

The next step will be to actually deprecate these: https://github.com/JuliaLang/julia/blob/99bf23db45241c10b6f2fd652e33aae60cad6059/base/boot.jl#L315-L320.  Running the tests with them deprecated will be an extra sanity check that you got them all.